### ISSUE TITLE
policystream - docker image doesn't run

### ISSUE THREAD

**Issue by thisisshi** [cloud-custodian/cloud-custodian#4087]
Pulling the latest version of policystream results in the following error:

```
$ docker pull cloudcustodian/policystream
$ docker run cloudcustodian/policystream
Traceback (most recent call last):
  File "/usr/local/bin/c7n-policystream", line 6, in <module>
    from policystream import cli
  File "/usr/local/lib/python3.7/dist-packages/policystream.py", line 21, in <module>
    from dateutil.tz import tzoffset, tzutc
  File "/usr/local/lib/python3.7/dist-packages/dateutil/tz/__init__.py", line 2, in <module>
    from .tz import *
  File "/usr/local/lib/python3.7/dist-packages/dateutil/tz/tz.py", line 19, in <module>
    import six
ModuleNotFoundError: No module named 'six'
```

I'm thinking this may be related to the change in switching the user from root to custodian as that's the most recent change to the dockerfile, during the pycon sprints I was able to confirm that it was working when writing the docs

**Comment by thisisshi** [comment]
can confirm that building c7b480fa71e8a9d3a4c1493a5c7276cebbc8546c works fine (last commit before changing to the non-root user)

**Comment by kapilt** [comment]
seems odd, six should be a dependency for dateutil and botocore

**Comment by kapilt** [comment]
verified with dive that six isn't in the image for some reason.. adding it explicitly to the dependencies

**Comment by kapilt** [comment]
merged in #4136
